### ISSUE TITLE
[NEW MAP] ID Card photo and records photograph have no clothes

### ISSUE THREAD

**Issue by Mwahahahaha** [Aurorastation/Aurora.3#2826]
I've been going around asking different players if they had this issue and I've only come across a small handful of people.

What happens is whenever I walk through the autodrober, or if I ready up, the photo on the ID card will show a naked version of my character.

Now what I tend to do with one of my characters is tie up the hair in the mirror at the beginning of the round:
If I readied up for the round start, I'd be naked in the photo with untied hair, and nothing else but my satchel on my back.
If I latejoined on Odin, went through the autodrobe thing without tying up hair, I'd be naked in the photo, holding onto the personal possessions box, leather satchel worn on back, and have untied hair.
If I latejoined on Odin, tied up my hair in the mirror and then went through the autodrobe thing, I'd be naked in the photo, holding onto the personal possessions box, leather satchel worn on back, and have tied up ponytail hair.

I've asked other people and they've also had similar issues. It was first brought to my attention when someone posted a screenshot of the character's security records to which they were, as on the ID, naked.

For humiliating reference:

![image](https://user-images.githubusercontent.com/3950115/27531291-9bf08f08-5a8e-11e7-8251-0eb2eac3fc0a.png)

This particular character was made before Aurora II, during Exodus. (June 2016).

**Comment by Mwahahahaha** [comment]
Would also like to point out:
Occupation is set to roboticist on high.

I pressed the hide equipment thing once when I first noticed it then again to put dem clothes back.

I have black gloves, wallet, and pAI device in custom loadout.

**Comment by Mwahahahaha** [comment]
Waiting out on the Odin until you get spawned at cryo gives you an ID with the starting Odin equipment (grey jumpsuit).

**Comment by Fire-and-Glory** [comment]
Record photos are still broke.

![image](https://user-images.githubusercontent.com/19973802/28491896-f3342c84-6f4c-11e7-984a-e3ac5f8ceaeb.png)

Taken today.

**Comment by Lohikar** [comment]
This still an issue? Every record I checked on live seemed fine.

**Comment by Mwahahahaha** [comment]
I was on a couple of days ago and the ID photo seemed fine.
I'm not entirely sure if others may still be experiencing the issue though.

May close it soon otherwise. @Fire-and-Glory

**Comment by Fire-and-Glory** [comment]
Latest batch of victims:

![victims](https://user-images.githubusercontent.com/19973802/28671843-2b1fb2ec-7332-11e7-91d4-958c621b8b89.png)

It is still an issue for me.

**Comment by Mwahahahaha** [comment]
May God have mercy on their souls.

They should really consider filing a lawsuit against the company for this harassment.

**Comment by Mwahahahaha** [comment]
If it's anything of interest I've noticed how I had two servers on at the same time:
One on Aurora proper and one was up from my own Daemon which was just running (on invisible) a VORE Station build (I was curious to see what they had, okay. I've only been on runtime once on my own test server and that was it.) and I noticed something.

On Aurora I had the character setup open which had one of my characters up as I was developing their stuffs and records and flavour text and yadda yadda yadda. But no dialog boxes were open there.
I alt-tabbed back onto the VORE station one and I noticed how the setup preview there wasn't the one that was supposed to be there but rather an image of the one generated from Aurora's preview.
